src/oclif/hooks/init.ts

```diff
@@ -1,50 +1,53 @@
-#!/usr/bin/env node
-const initTimeStartTime = process.hrtime()
-
-import 'v8-compile-cache'
-
 import axios from 'axios'
 import chalk from 'chalk'
-import { all as clearCachedModules } from 'clear-module'
-import { CommandNotFoundError, find, MissingRequiredArgsError, run as unboundRun } from 'findhelp'
 import os from 'os'
-import path from 'path'
-import { without } from 'ramda'
-import * as pkg from '../package.json'
-import { CLIPreTasks } from './CLIPreTasks/CLIPreTasks'
-import * as conf from './conf'
-import { envCookies } from './env'
-import { CommandError, SSEConnectionError, UserCancelledError } from './errors'
-import { Token } from './lib/auth/Token'
-import { TelemetryCollector } from './lib/telemetry/TelemetryCollector'
-import log from './logger'
-import tree from './modules/tree'
-import { checkAndOpenNPSLink } from './nps'
-import notify from './update'
-import { isVerbose, VERBOSE } from './utils'
-import { Metric } from './lib/metrics/MetricReport'
-import { hrTimeToMs } from './lib/utils/hrTimeToMs'
-
-const run = command => Promise.resolve(unboundRun.call(tree, command, path.join(__dirname, 'modules')))
+import { all as clearCachedModules } from 'clear-module'
+import { CommandNotFoundError, MissingRequiredArgsError } from 'findhelp'
```

Comment:
I think this can be removed since oclif deals with this errors. `findhelp` can be removed from `package.json` too